### ISSUE TITLE
Extensions: announce state of extension installation

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52664]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Extensions.
3. Type something in the Search Textbox and tab till install button.

**Actual:**
When we Click on install button, the text above the button changes..i.e.(installing) which is not read by the screenreader and after installing, a Reload button appears and also a icon which is not informed by the screenreader and after pressing tab key focus does not move on the text which has installed,it goes somewhere else.

**Expected:**
NVDA should announce some information about the reload button and should also read the "installing" when it is getting installed and focus should go on the very first item which has got installed. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know about the controls which has appeared after installing the particular extensions.

Attachment for Reference: 
[A11y VSCode Usability Extensions Install button issue.pptx](https://github.com/Microsoft/vscode/files/2126641/A11y.VSCode.Usability.Extensions.Install.button.issue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@ramya-rao-a do we plan to tackle this in September? It would be great since we depend on this one for a better accessibility score?

**Comment by ramya-rao-a** [comment]
Yes, this is for September

**Comment by ramya-rao-a** [comment]
Note to verifier: Also test cases of uninstall and update and ensure that appropriate text is read out. Ensure that the need to reload VS Code is also mentioned
